CONTRIBUTING.md

```diff
@@ -123,16 +127,16 @@ be obtained quite easily.
 
 The process of getting community commit access is as follows:
 
-1. **Email the developer list:** This first step is all about communication. In order to grant commit access the other developers on the project must first know what the intention is. Therefore any developer looking for commit access must first describe what they want to commit (usually a community module), and what it does.
+1. **Email the developer list:** This first step is all about communication. In order to grant commit access, the other developers on the project must first know what the intention is. Therefore any developer looking for commit access must first describe what they want to commit (usually a community module), and what it does.
 
 2. **Sign up for a GitHub account:** GeoServer source code is hosted on Github and you'll need an account in order to access it. You can sign-up [here](https://github.com/signup/).
 
 3. **Print, sign, scan and send the contributor agreement:**
 
-   * [corporate_contributor.pdf](https://www.osgeo.org/wp-content/uploads/corporate_contributor.pdf)
-   * [individual_contributor.pdf](https://www.osgeo.org/wp-content/uploads/individual_contributor.pdf)
+   - [corporate_contributor.pdf](https://www.osgeo.org/wp-content/uploads/corporate_contributor.pdf)
+   - [individual_contributor.pdf](https://www.osgeo.org/wp-content/uploads/individual_contributor.pdf)
 
-   Scanned assignment agreement can be emailed to [info@osgeo.org](mailto:info@osgeo.org) at OpenPlans.
+   A scanned assignment agreement can be emailed to [info@osgeo.org](mailto:info@osgeo.org) at OpenPlans.
```

Comment:
I note this should say "OSGeo" rather than "OpenPlans" as the copyright has been transferred to Open Source Geospatial Foundation.